### DISCUSSION TITLE
Label of visualization is not actual

### DISCUSSION THREAD

**Discussion by esp-thaihoang** [getredash/redash#6993] [Q&A]
I have query database SQL to count total each utm_source and group by created_at.
The table result show actually data. But I when render visualization, it shows value not correct with table result. Like image
For example, on 01/03/2024 (DD/MM/YYYY), table result shows 
<img width="424" alt="image" src="https://github.com/getredash/redash/assets/168699144/a1535371-d564-4e04-8274-db1f598e417f">
but visualization stack chart show 
<img width="168" alt="image" src="https://github.com/getredash/redash/assets/168699144/09f8b6ad-41f7-4f26-826e-88da4e02f92a">

How to fix it?

Thanks for helps

**Comment by gaecoli** [comment]
<img width="397" alt="image" src="https://github.com/getredash/redash/assets/46274612/44295e6c-96a7-4050-ab4c-9c0777535b25">
